controllers/tasksRequests.js

```diff
@@ -23,6 +23,26 @@ const fetchTaskRequests = async (_, res) => {
   }
 };
 
+const fetchTaskRequestById = async (req, res) => {
+  try {
+    const id = req.params.id;
+    const data = await taskRequestsModel.fetchTaskRequestById(id);
+
+    if (!data.taskRequestExists) {
```

Comment:
here data object returned from the taskRequestsModel.fetchTaskRequestById() method contains a property called taskRequestExists. This property is true if the task request was found, and false if it was not found. However, the code you provided does not check the value of this property before returning the response. This means that if the task request was not found, the code will still return a status code of 200

Reply:
I am specifically testing for `taskRequestExists` key's value, and I have also written a test for the same, which you can find in #1300

Reply:
@RitikJaiswal75 can you look here please

Reply:
I cant find this test in #1300 (looking for it in integration test)

Reply:
@RitikJaiswal75 Can you specify what exactly are you looking for